cross/json-c/Makefile

```diff
@@ -0,0 +1,24 @@
+PKG_NAME = json-c
+PKG_VERS = 0.12-20140410
```

Comment:
Notice a more recent version is available and expected for introduction from another PR #3873 
https://github.com/SynoCommunity/spksrc/blob/364de55b69714ed8c806caa38953064b5e7efe55/cross/json-c/Makefile